core-lightning/docker-compose.yml

```diff
@@ -5,55 +5,49 @@ services:
     environment:
       APP_HOST: $APP_CORE_LIGHTNING_IP
       APP_PORT: $APP_CORE_LIGHTNING_PORT
-
+    container_name: core-lightning_app_proxy_1
```

Comment:
Can you please remove the `container_name` line from each of the services (app_proxy, app, lightningd, tor)? These get dynamically generated by umbrelOS and added to a user's installed compose file on app install so we don't want to hard-code them here.

Reply:
Removed all container names.